common/network-common/pom.xml

```diff
@@ -45,6 +45,22 @@
       <artifactId>commons-lang3</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.fusesource.leveldbjni</groupId>
```

Comment:
I think network-yarn depends on network-shuffle, so the utility class can be put into network-shuffle.